utils/utils.go

```diff
@@ -59,6 +59,22 @@ func FuzzyAgo(ago time.Duration) string {
 	return fmtDuration(int(ago.Hours()/24/365), "year")
 }
 
+func FuzzyAgoAbbr(now time.Time, createdAt time.Time) string {
```

Comment:
@ampinsk and I thought it would be nice to start using shorter time output. We decided to just start with comment created at date for now and see how that goes. We can decide to change the other use cases of `FuzzyAgo` to use this new abbreviated version if we like it.

Reply:
Great idea! 👏